Gemfile

```diff
@@ -117,8 +117,8 @@ end
 #
 unless ENV["APPLIANCE"]
   group :development do
-    gem "haml_lint",        "~>0.18.0", :require => false
-    gem "rubocop",          "~>0.37.2", :require => false
+    gem "haml_lint",        "~>0.20.0", :require => false
```

Comment:
As discussed in gitter, let's move this to manageiq-ui-classic.  It's up to you if you want to do it in this PR.